src/Form/Form.js

```diff
@@ -149,11 +149,11 @@ class Form {
         return this
     }
 
-    areAllInputsEmpty () {
+    areAllInputsEmpty (inputType) {
         let allEmpty = true
         this.execOnInputs((input) => {
             if (input.value) allEmpty = false
-        })
+        }, inputType)
```

Comment:
We're now passing the input type to check only specific input types. Solves a bug where the tooltip might not open if other parts of the form are filled up.